wallet-new/src/Cardano/Wallet/API/V1/Handlers/Wallets.hs

```diff
@@ -91,3 +93,21 @@ updateWallet pwl wid walletUpdateRequest = do
     case res of
          Left e  -> throwM e
          Right w -> return $ single w
+
+checkExternalWallet :: PassiveWalletLayer IO
+                    -> PublicKeyAsBase58
+                    -> Handler (WalletResponse WalletAndTxHistory)
+checkExternalWallet _encodedRootPK =
+    error "Cardano Hardware Wallet feature, check external wallet, unimplemented yet."
+
+newExternalWallet :: PassiveWalletLayer IO
+                  -> NewExternalWallet
+                  -> Handler (WalletResponse Wallet)
+newExternalWallet _newExtWallet =
+    error "Cardano Hardware Wallet feature, new external wallet, unimplemented yet."
+
+deleteExternalWallet :: PassiveWalletLayer IO
+                     -> PublicKeyAsBase58
+                     -> Handler NoContent
+deleteExternalWallet _encodedRootPK =
+    error "Cardano Hardware Wallet feature, delete external wallet, unimplemented yet."
```

Comment:
Do we have tickets for that? If not, can we create one and assign Squad 1 or Squad 2 to it :) ?